README.md

```diff
@@ -697,6 +697,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Java, SQL, and jOOQ](http://blog.jooq.org/)
 * [Java.net](http://java.net/)
 * [Javalobby](http://java.dzone.com/)
+* [JavaTutorial](http://docs.oracle.com/javase/tutorial/)
```

Comment:
We shouldn't add just every ressource. I think that people who find this list, want to experience new and exciting libraries and wouldn't need a Java SE tutorial.